### PR TITLE
Show basket prodcuts with a ReferenceField

### PR DESCRIPTION
**Pull request by adguernier**
- [x] ~~use a ReferenceArrayField to show products related to~~
- [x]  use a ArrayField to show products related to
- [x] show quantity and total price, based on the basket lines
- [x] create a custom view
image

### CODE REVIEW

**Review by djhi — APPROVED** [review]

File: src/ui/order/Basket.tsx [hunk 1]
```diff
@@ -1,71 +1,49 @@
...
-  const { data: products, isLoading } = useGetMany("products", {
-    ids: productIds,
-  });
-
-  if (isLoading) return <Loading />;
+  return (
+    <ArrayField source="basket">
+      <Datagrid
+        bulkActionButtons={false}
+        rowClick={(id, resource, record) => `/products/${record.product_id}`}
```
Comment (djhi) [inline src/ui/order/Basket.tsx]: You should probably disable sorting too

**Review by djhi — CHANGES_REQUESTED** [review]

**Review by djhi — CHANGES_REQUESTED** [review]
And as we discussed, isn't the Datagrid overkill here (sorting, pagination, etc)? Maybe you should actually try building a custom table view with `useListContext`

**Review by fzaninotto — APPROVED** [review]

**Comment by adguernier** [comment]
> And as we discussed, isn't the Datagrid overkill here (sorting, pagination, etc)? Maybe you should actually try building a custom table view with `useListContext`

Yes I can try

**Comment by djhi** [comment]
Can you add a screenshot?

**Review by djhi — APPROVED** [review]

**Pull request merged**